org_qubes_os_initial_setup/ks/qubes.py

```diff
@@ -269,13 +291,37 @@ def configure_default_kernel(self):
         self.run_command([
             '/usr/bin/qubes-prefs', 'default-kernel', default_kernel])
 
+    def configure_default_pool(self):
+        self.set_stage("Setting up default pool")
+        if self.vg_tpool:
+            self.run_command(['/usr/bin/qvm-pool', '--add', 'default', 'lvm_thin',
+                              '-o', 'volume_group=%s,thin_pool=%s,revisions_to_keep=2' % self.vg_tpool])
```

Comment:
What about actually setting it as the default pool?

Reply:
Yes we can enforce this more but looking at the code of core-admin, it sets default pool on multiple search and a pool called 'default' is the first one. So maybe better adding the pool with name 'vm' and set it as default?